util/filter.lua

```diff
@@ -154,7 +154,7 @@ M.filters = {
         validator = function(amount)
             return function(auction_record)
                 local item_info = info.item(auction_record.item_id)
-                local disenchant_value = item_info and disenchant.value(item_info.item_id, item_info.slot, item_info.quality, item_info.level)
+                local disenchant_value = item_info and disenchant.value(auction_record.item_id, item_info.slot, item_info.quality, item_info.level)
```

Comment:
Why was this necessary, shouldn't it be the same id?

Reply:
Because the `Item_info` table does not have the `item_id` field. Therefore, in `disenchant.value`, the first argument has always had the value of `nil`.

Reply:
Ah yes, you're right, I thought it did.